lib/giraph/schema.rb

```diff
@@ -0,0 +1,46 @@
+module Giraph
+  # GraphQL::Schema wrapper allowing decleration of
+  # resolver objects per op type (query or mutation)
+  class Schema < GraphQL::Schema
```

Comment:
I agree in general but in this case the subclass is not relying on anything that's in the base class. Not methods called (apart from `super` calls which I will have to do with a delegate as well), no properties used. Even if the whole implementation changed, as long as `execute` takes a `query` string first and named arguments rest, we're good. We'll be in the same boat with a delegator too, `execute`-wise.

Not sure if one is better than the other in that case.